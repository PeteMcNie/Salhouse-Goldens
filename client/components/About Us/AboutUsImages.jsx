import React from 'react'

import Image from 'react-bootstrap/Image'
import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'

function AboutUsImages () {
  return (
    <div>
      <Container>
        <Row>
          <Col xs={12} md={6} lg={4}>
            <Image src="images/au1.jpg" thumbnail fluid />
            <p>Phoenix poking her tongue out.</p>
          </Col>
          <Col xs={12} md={6} lg={4}>
            <Image src="images/au2.jpg" thumbnail fluid />
            <p>Kula putting Shadow&apos;s leg to the taste test!</p>
          </Col>
          <Col xs={12} md={6} lg={4}>
            <Image src="images/au3.jpg" thumbnail fluid />
            <p>Shadow in her favourite &ldquo;car crash&ldquo; position!</p>
          </Col>
        </Row>
      </Container>
    </div>

  )
}

export default AboutUsImages
