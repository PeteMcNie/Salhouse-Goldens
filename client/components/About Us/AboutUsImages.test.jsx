import React from 'react'
import { render } from '@testing-library/react'

import AboutUsImages from './AboutUsImages'

test('AboutUs component to match snapshot', () => {
  const { container } = render(<AboutUsImages />)
  expect(container).toMatchSnapshot()
})
