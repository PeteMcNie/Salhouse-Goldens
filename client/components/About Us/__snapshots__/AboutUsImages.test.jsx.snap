// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs component to match snapshot 1`] = `
<div>
  <div>
    <h3>
      Photos
    </h3>
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-md-4 col-6"
        >
          <img
            class="img-fluid img-thumbnail"
            src="images/au1.jpg"
          />
          <p>
            Phoenix poking her tongue out.
          </p>
        </div>
        <div
          class="col-md-4 col-6"
        >
          <img
            class="img-fluid img-thumbnail"
            src="images/au2.jpg"
          />
          <p>
            Kula putting Shadow's leg to the taste test!
          </p>
        </div>
        <div
          class="col-md-4 col-6"
        >
          <img
            class="img-fluid img-thumbnail"
            src="images/au3.jpg"
          />
          <p>
            Shadow in her favourite “car crash“ position!
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
